umh-core/pkg/fsm/redpanda/reconcile.go

```diff
@@ -399,15 +399,15 @@ func (r *RedpandaInstance) reconcileTransitionToStopped(ctx context.Context, ser
 			return err, false
 		}
 		// Send event to transition to Stopping
-		r.PreviousObservedState.ServiceInfo.StatusReason = "stopping"
+		r.PreviousObservedState.ServiceInfo.StatusReason = "Redpanda stopping"
```

Comment:
i dont think this is the correct place to change it. why not simply change it in the callers? so in protocol converters for example